src/AppInstallerRepositoryCore/Rest/RestClient.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "RestClient.h"
+#include "Rest/Schema/1_0/Interface.h"
+
+using namespace AppInstaller::Repository::Rest::Schema;
+
+namespace AppInstaller::Repository::Rest
+{
+    RestClient::RestClient(std::string restApi)

```

Comment:
>std::string restApi [](start = 27, length = 19)

I think const & here
I followed through the constructors code, I did not see a big benefit for using copy here then std::move below

Reply:
The benefit of moving it higher in the stack is that the caller might have a value that they can move in.  If you can't take that value, that is one more string copy that is needed.

---
In reply to: [594708888](https://github.com/microsoft/winget-cli/pull/765#discussion_r594708888) [](ancestors = 594708888)